tap_typeform/streams.py

```diff
@@ -136,6 +136,23 @@ def get_landings(atx, form_id, bookmark):
 
         yield from items
 
+def fetch_sub_questions(row):
+    '''This function fetches records for each sub_question in a question group and returns a list of fetched sub_questions'''
+    sub_questions = [] #Creating blank list to accommodate each sub-question's records
+
+    #If question group has no sub_questions, return blank list
+    if row['properties'].get('fields') == None :
+        return None
+
+    #Appending each sub-question to the list
+    for question in row['properties']['fields']:
+        sub_questions.append({
+            "question_id": question['id'],
+            "title": question['title'],
+            "ref": question['ref']
+            })
```

Comment:
```suggestion
    #Appending each sub-question to the list
    for question in row['properties'].get('fields'):
        sub_questions.append({
            "question_id": question['id'],
            "title": question['title'],
            "ref": question['ref']
            })
```
We can remove the if condition because for loop will not iterate on `None or []` so no need to handle it.